src/oc/storage/api/boards.clj

```diff
@@ -89,19 +89,21 @@
       (assoc :entries entry-reps)))))
 
 ;; ----- Validations -----
+
 (defn- valid-entry-with-board?
   [conn entry author]
-  (if-let* [entry-uuid (:uuid entry)
-           found-entry (entry-res/get-entry conn entry-uuid)]
-    (merge found-entry entry)
+  (if-let* [status (or (:status entry) "published")
```

Comment:
why is the default `published` here?

Reply:
Hmm, it looks like this is old code.  Still unsure why the default is published :)

Reply:
I think this is old code that pre-dates the migration that gave everything that existed up to that point a default value for status. In theory we don't have any entries w/o a status now so I think it's OK to drop the `or`.

Reply:
Done